src/schemas/paths.js

```diff
@@ -109,48 +112,76 @@ export const paths = {
             'label': 'GET',
             'hideValueIfEmpty': true,
             'isCollapsible': false,
+            'overrideI18nPath': 'form.paths.item.operation',
```

Comment:
Could we just call this `i18nPath`? It would be a bit shorter, and easier to read. Is `i18nPath` already used?